quantum/process_keycode/process_leader.c

```diff
@@ -68,9 +68,7 @@ bool process_leader(uint16_t keycode, keyrecord_t *record) {
     } else {
       if (keycode == KC_LEAD) {
         qk_leader_start();
-        return false;
```

Comment:
This should be left here, as it tells the firmware that "it's done processing `KC_LEAD` and doesn't need to be handled anymore". 

Though, it could be removed without much issue.

However, the "break" needs to be removed, and that's my fault that it's there. Sorry.